opengemini-client-api/src/main/java/io/opengemini/client/api/RetentionPolicy.java

```diff
@@ -0,0 +1,24 @@
+package io.opengemini.client.api;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class RetentionPolicy {
+    String name;
+
+    String duration;
+
+    String shardGroupDuration;
+
+    String hotDuration;
+
+    String warmDuration;
+
+    String indexDuration;
+
+    int replicaNum;
+
+    boolean isDefault;
```

Comment:
consider default？

Reply:
default might be a keyword in most language. So I would approve it.

Reply:
```default``` is a reserved keyword in Java. It is always using with ``` switch / case ```